docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -43,55 +38,43 @@ Additionally, if you wish to support older targets, you will need to install tar
     
-    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet54`.
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `netstandard1.3`.
 
-    ```javascript
+    ```json
     {
         "frameworks":{
-            "dotnet54":{}
+            "netstandard1.3":{}
         }
     }
     ```
     
-    For example, you need to target `dotnet51` to support Windows Phone Silverlight 8. The .NET Platform Standard versions in a backward-compatible way. That means that `dotnet51` libraries run on `dotnet52` platforms and later (e.g. .NET Framework 4.6 and .NET Core). When targeting `dotnet51`, you get the benefit of more platform reach than `dotnet52` but have access to fewer APIs. Each .NET Platform Standard version adds APIs and drops platforms. You should select the Standard version that has the right mix of APIs and platform support for your needs.
+    The .NET Platform Standard versions in a backward-compatible way. That means that `netstandard1.0` libraries run on `netstandard1.1` platforms and higher. When targeting `netstandard1.0`, you get the benefit of more platform reach than `netstandard1.1` but have access to fewer APIs. Each .NET Platform Standard version adds APIs and drops platforms. You should select the Standard version that has the right mix of APIs and platform support for your needs.
     
-3. If you want to target the .NET Framework versions 4.0 or below, or you wish to use an API available in .NET Framework but not in .NET Core (e.g. `System.Drawing`), you'll need to read the next sections.
+3. If you want to target the .NET Framework versions 4.0 or below, or you wish to use an API available in .NET Framework but not in .NET Core (e.g. `System.Drawing`), you'll need to read the following sections.
 
 ## How to target .NET Framework
 
 **NOTE:** These instructions assume you have .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
 
 Keep in mind that some of the .NET Framework versions used here are no longer in support.  [Refer to this FAQ](https://support.microsoft.com/gp/framework_faq/en-us) about unsupported versions.
 
-You may want to reach the maximum developers and projects, so use .NET Framework 4 as your baseline target.  From a project targeting perspective, this is quite simple!  You will need to specifically target the version of .NET Framework with its relevant Target Framework Moniker.  These are mapped out here:
+You may want to reach the maximum developers and projects, so use .NET Framework 4 as your baseline target.  To target .NET Framework, you will need to begin by using the correct Target Framework Moniker (TFM) that corresponds to the .NET Framework version you wish to support.
```

Comment:
You might want to say:
If you want to reach the maximum developers and projects, use the .NET Framework 4 as your baseline target. To target the .NET Framework, ...